bodhi/client/__init__.py

```diff
@@ -236,6 +236,8 @@ def edit(user, password, url, **kwargs):
               help='Query only critical path packages')
 @click.option('--cves', help='Query by CVE id')
 @click.option('--packages', help='Query by package name(s)')
+@click.option('--content-type', help='Query updates based on content type',
+              type=click.Choice(['rpm', 'module']))  # And someday, container.
```

Comment:
Hm, a thought.  If the default is anything other than `None` (e.g., if it is 'rpm'), then there is no way to query for *all* updates of *any* content type.